### PR TITLE
feature/st 233 form field ut

### PR DESCRIPTION
**Pull request by pgs-gmartowski**

### CODE REVIEW

File: test/core/shared/FormField.spec.ts [hunk 1]
```diff
@@ -1,107 +1,139 @@
...
-      expect(calls[0][0]).toBe('focus');
-      expect(calls[0][1]).toBeInstanceOf(Function);
+      expect(instance._inputElement.getAttribute('maxlength')).toEqual(inputAttributes.maxlength.toString());
+      // @ts-ignore
+      expect(instance._inputElement.getAttribute('minlength')).toEqual(inputAttributes.minlength.toString());
     });
   });
+  // given
+  describe('_setInputListeners()', () => {
+    const { instance } = FormFieldFixture();
```
Comment (pgs-gmartowski) [inline test/core/shared/FormField.spec.ts]: info: there are some problems with mocking different events in this one function - I just leave it for a moment and go back later

File: src/core/integrations/ApplePay.ts [hunk 2]
```diff
@@ -187,7 +187,9 @@ export class ApplePay {
       ? (this.buttonText = buttonText)
       : (this.buttonText = ApplePay.AVAILABLE_BUTTON_TEXTS[0]);
 
-    this._applePayButtonProps.style = `-webkit-appearance: -apple-pay-button; -apple-pay-button-type: ${this.buttonText}; -apple-pay-button-style: ${this.buttonStyle}`;
+    this._applePayButtonProps.style = `-webkit-appearance: -apple-pay-button; -apple-pay-button-type: ${
+      this.buttonText
+    }; -apple-pay-button-style: ${this.buttonStyle}`;
```
Comment (pgs-gmartowski) [inline src/core/integrations/ApplePay.ts]: possibly some IDE formatting difference

File: test/core/integrations/VisaCheckout.spec.ts [hunk 3]
```diff
@@ -510,7 +510,9 @@ function VisaCheckoutFixture() {
   };
   const sandboxAssets = {
     sdk: 'https://sandbox-assets.secure.checkout.visa.com/checkout-widget/resources/js/integration/v1/sdk.js',
-    buttonImg: `https://sandbox.secure.checkout.visa.com/wallet-services-web/xo/button.png?color=${config.buttonSettings.color}&size=${config.buttonSettings.size}`
+    buttonImg: `https://sandbox.secure.checkout.visa.com/wallet-services-web/xo/button.png?color=${
+      config.buttonSettings.color
+    }&size=${config.buttonSettings.size}`
```
Comment (pgs-gmartowski) [inline test/core/integrations/VisaCheckout.spec.ts]: possibly some IDE formatting difference

File: test/core/shared/FormField.spec.ts (hunk 1 above)
Comment (james-simpson-ST) [reply test/core/shared/FormField.spec.ts]: Okay

File: test/core/integrations/VisaCheckout.spec.ts (hunk 3 above)
Comment (james-simpson-ST) [reply test/core/integrations/VisaCheckout.spec.ts]: Yes this changed on mine too, no problem

**Pull request merged**